apis/orgpolicy/v1alpha1/types.generated.go

```diff
@@ -71,6 +70,17 @@ type PolicySpec_PolicyRule struct {
 	//  'tagValues/456')".
 	// +kcc:proto:field=google.cloud.orgpolicy.v2.PolicySpec.PolicyRule.condition
 	Condition *Expr `json:"condition,omitempty"`
+
+	// Optional. Required for managed constraints if parameters are defined.
+	//  Passes parameter values when policy enforcement is enabled. Ensure that
+	//  parameter value types match those defined in the constraint definition.
+	//  For example:
+	//  {
+	//    "allowedLocations" : ["us-east1", "us-west1"],
+	//    "allowAll" : true
+	//  }
+	// +kcc:proto:field=google.cloud.orgpolicy.v2.PolicySpec.PolicyRule.parameters
+	Parameters map[string]string `json:"parameters,omitempty"`
```

Comment:
Looks like the type here is not `map[string]string`

Reply:
Good point! I trusted the type generator and didn't notice the type can actually be non-string.